elementary/gtk-3.0/granite-widgets-dark.css

```
@@ -88,6 +88,45 @@
     box-shadow: none;
 }
 
+/**************
+* Storage Bar *
+**************/
+
+.storage-bar .fill-block {
+    background-image:
+        linear-gradient(
+            to bottom,
+            @BLACK_300,
+            @BLACK_500
+        );
+    box-shadow:
+        inset 0 0 0 1px alpha (#ccc, 0.05),
```

In any new stylesheet code, we've been omitting the space. Code syntax highlights properly without the space, plus it's more consistent with built-in CSS methods and languages like Sass.

```suggestion
        inset 0 0 0 1px alpha(#ccc, 0.05),
```

---

I'll go ahead and fix that, but this is also the case in https://github.com/elementary/stylesheet/blob/master/elementary/gtk-3.0/granite-widgets.css#L274 where I copied this from - Should I fix this here too or make a new PR for it?